ocean_lib/ocean/ocean_assets.py

```diff
@@ -303,7 +311,7 @@ def create(
                 del file["url"]
             metadata_copy["encryptedFiles"] = files_encrypted
         else:
-            raise AssertionError("Encrypting the files failed.")
+            raise DDOError("Encrypting the files failed.")
```

Comment:
Should rename away from **location** to **type** of error. 

The closest thing I found in other python repos was "UnicodeEncodeError". But that's not quite right.

Therefore here I suggest a custom class of "EncryptionError".